app/controllers/company/opening.js

```diff
@@ -42,6 +42,24 @@ function newOpening(req, res, next) {
   });
 }
 
+/**
+ * delete  opening
+ * @param  {Object}   req  Request Object
+ * @param  {Object}   res  Response Object
+ * @param  {Function} next Function to pass control to the next middleware
+ */
+
+function deleteOpening(req, res, next) {
+  openingLib.deleteOpening(req.user, req.body, function(err, fetchedInstance) {
```

Comment:
This function is wrong `deleteOpening function` in lib is having only two parameters but you are passing 3 parameters.
Check and run API.